### PR TITLE
3.3 version OHOS adaptation

### PR DESCRIPTION
**Pull request by IceBBBBB**

### CODE REVIEW

File: cmake/Modules/CocosBuildHelpers.cmake [hunk 1]
```diff
@@ -139,6 +139,8 @@ elseif (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     set(APPLE TRUE)
     set(SYSTEM_STRING "Mac OSX")
   endif()
+elseif(OHOS)
+  set(SYSTEM_STRING "OpenHarmony")
```
Comment (minggo) [inline cmake/Modules/CocosBuildHelpers.cmake]: Should change to `HarmonyOS Next`?

File: tests/lua-tests/src/controller.lua [hunk 2]
```diff
@@ -1,4 +1,7 @@
 
+-- jit off
+local jit = require("jit")
+jit.off()
```
Comment (minggo) [inline tests/lua-tests/src/controller.lua]: Why have to disable jit?

File: cmake/Modules/CocosBuildHelpers.cmake (hunk 1 above)
Comment (IceBBBBB) [reply cmake/Modules/CocosBuildHelpers.cmake]: modified

File: tests/lua-tests/src/controller.lua (hunk 2 above)
Comment (IceBBBBB) [reply tests/lua-tests/src/controller.lua]: Due to security concerns, JIT needs to be turned off；We will consider adapting the third-party library to disable JIT in the future

Comment (minggo) [reply tests/lua-tests/src/controller.lua]: Ok, then it is better to only disable on OHOS and do it in mainmenu.lua.

Comment (IceBBBBB) [reply tests/lua-tests/src/controller.lua]: Due to security concerns, JIT needs to be turned off；We will consider adapting the third-party library to disable JIT in the future

Comment (minggo) [reply tests/lua-tests/src/controller.lua]: But current implementation disable it for all platforms.

Comment (IceBBBBB) [reply tests/lua-tests/src/controller.lua]: sorry，modified

**Pull request merged**